.github/workflows/build-check-install.yaml

```diff
@@ -1,222 +0,0 @@
-          header: R CMD check for R v${{ matrix.config.tag }}
-          message: |
-            #### R CMD check for R v${{ matrix.config.tag }}
-
-            `R CMD check` resulted in `${{ steps.check.outputs.check-status }}`
-
-            <details><summary>Click here to show full output</summary>
-
-            ```R
-            ${{ steps.check.outputs.check-output }}
-            ```
-
-            </details>
-
-            *Pusher: @${{ github.actor }}, Action: `${{ github.event_name }}`*
-
-
-      - name: Catch warnings in R CMD check output
-        id: catch-errors
-        run: |
-          x <- tail(readLines("${{ github.event.repository.name }}.Rcheck/00check.log"), 1)
-          if (!grepl("^Status", x)) stop("No status line found in R CMD check log")
-          if (grepl("ERROR", x)) stop("R CMD check has errors")
-          if (grepl("WARNING", x)) stop("R CMD check has warnings")
-        shell: Rscript {0}
-
-      - name: Upload check results
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.config.tag }}-results
-          path: ${{ github.event.repository.name }}.Rcheck/00check.log
-
-      - name: Check whether JUnit XML report exists
-        id: check_junit_xml
-        uses: andstor/file-existence-action@v1
-        with:
-          files: "${{ github.event.repository.name }}.Rcheck/tests/testthat/junit-result.xml"
-
-      - name: Publish Unit Test Summary
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        if: ${{ steps.check_junit_xml.outputs.files_exists == 'true' && github.event_name == 'pull_request' }}
-        with:
-          check_name: Unit Tests Summary
-          files: "${{ github.event.repository.name }}.Rcheck/tests/testthat/junit-result.xml"
-
-      - name: Convert JUnit XML Test Results to Markdown
-        id: xml2md
-        if: ${{ steps.check_junit_xml.outputs.files_exists == 'true' && github.event_name == 'pull_request' }}
-        run: |
-          wget -q https://github.com/dinakar29/junit2md/releases/download/v0.1.0/junit2md_0.1.0_linux-amd64 -O /tmp/junit2md
-          chmod +x /tmp/junit2md
-          output="$(/tmp/junit2md ${{ github.event.repository.name }}.Rcheck/tests/testthat/junit-result.xml)"
-          output="${output//'%'/'%25'}"
-          output="${output//$'\n'/'%0A'}"
-          output="${output//$'\r'/'%0D'}"
-          echo "::set-output name=test-results-md::$output"
-        shell: bash
-
-      - name: Post Unit Test Details
-        uses: marocchino/sticky-pull-request-comment@v2
-        if: ${{ steps.check_junit_xml.outputs.files_exists == 'true' && github.event_name == 'pull_request' }}
-        with:
-          header: Unit Test Details
-          message: |
-            ## Unit Test Details
-
-            ${{ steps.xml2md.outputs.test-results-md }}
-
-      - name: Install R package
-        run: R CMD INSTALL ${{ env.PKGBUILD }}
-        shell: bash
-
-      - name: E2e package functionality
```

Comment:
@dinakar29 - this is a special e2e staged.deps system test which we should keep which just requires the package installed -> can you refactor this into its own workflow

Reply:
Yes definitely.